portfolio/src/main/webapp/script.js

```diff
@@ -27,23 +33,14 @@ function addRandomQuote() {
       'Give up? Give up?! The day may come when we’ll give up on fruitless searches after a mere 11 minutes, but that day is NOT today! The day may come when our favorite reptile may be lost from our memories and his enduring love of mushrooms forgotten, but that day is not today! Today we search! We will search for him in the streets, we will search for him in the trenches, we will search for him in the alleys and the mini-malls and the cul-de-sacs of this fair land. We will search for him in the multi-level car parks and municipal recreation facilities. And we few, we happy few, we small band of brothers - and girl from across the street. We shall not cease ’til he is found! - Ferb (Phineas and Ferb)', 
       'I’m just the keeper of secrets. Like the Vatican. - Jamal (On My Block)'];
 
-  // Pick a random quote.
+  // Picks random quote
   const quote = quotes[Math.floor(Math.random() * quotes.length)];
```

Comment:
ideally, we should use a int cast here.

Reply:
Why should I int cast? Also, I think this code was provided by SPS.

Reply:
It seems an implicit way to typecast the underlying floating point value to an int, we generally prefer to have explicit casts.